### PR TITLE
Replace Travis CI with GitHub Actions

### LINKED ISSUE #27: Move from Travis CI to GitHub Actions?
Would you be interested in a pull request that moves CI from Travis CI to GitHub Actions, or at least adds GitHub Actions alongside Travis CI? It seems travis-ci.org will stop being useful from Dec 31 2020. I can do it, except that I think you need to create the workflow yml, first, in `master`, after which I can branch off of.

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
Closes #27.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# rebar_cmd [![Build Status](https://travis-ci.org/gootik/rebar_cmd.svg?branch=master)](https://travis-ci.org/gootik/hakicache) [![Hex.pm](https://img.shields.io/hexpm/v/rebar_cmd.svg)](https://hex.pm/packages/rebar_cmd)
```
Comment (paulo-ferraz-oliveira) [inline README.md:1]: `hakicache` was an error here 😄

Comment (gootik) [reply README.md:1]: HAHA! oops

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      - master
+jobs:
+  ci:
+    name: Run checks and tests over ${{matrix.otp_vsn}} and ${{matrix.os}}
+    runs-on: ${{matrix.os}}
+    container:
+      image: erlang:${{matrix.otp_vsn}}
+    strategy:
+      matrix:
+        otp_vsn: [19.3, 20.3, 21.3, 22.3, 23.0]
```
Comment (gootik) [inline .github/workflows/ci.yml:18]: let's add 23.1 as well

Comment (paulo-ferraz-oliveira) [reply .github/workflows/ci.yml:18]: ... it's already merged 😄

**Pull request merged**